backend/app/api/endpoints/admin.py

```
@@ -0,0 +1,427 @@
+    except Exception as e:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Failed to create investor: {str(e)}",
+        )
+
+
+@router.post("/investors/bulk-import")
+async def bulk_import_investors(
+    session: SessionDep,
+    current_user: CurrentUser,
+    file: UploadFile = File(...),
```

Ruff (0.11.9)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)